<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWord</class>
 <widget class="QDialog" name="AddWord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a new word</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTextEdit" name="leftTextEdit">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="rightTextEdit">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="addButton">
             <property name="text">
              <string/>
             </property>
             <property name="shortcut">
              <string>Ctrl+Return</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearLinesButton">
             <property name="text">
              <string/>
             </property>
             <property name="shortcut">
              <string>Ctrl+Shift+D</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearForeignButton">
             <property name="text">
              <string/>
             </property>
             <property name="shortcut">
              <string>Ctrl+L</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearRusButton">
             <property name="text">
              <string/>
             </property>
             <property name="shortcut">
              <string>Ctrl+R</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fontUpButton">
             <property name="text">
              <string/>
             </property>
             <property name="shortcut">
              <string>Ctrl+U</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fontDownButton">
             <property name="text">
              <string/>
             </property>
             <property name="shortcut">
              <string>Ctrl+D</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>288</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="modeComboBox">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="selectButton">
           <property name="text">
            <string/>
           </property>
           <property name="shortcut">
            <string>Ctrl+Enter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
